release.config.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
     '@semantic-release/release-notes-generator',
     '@semantic-release/changelog',
     '@semantic-release/github',
+    {
+      message: 'Docs: ${nextRelease.version} [skip ci]\n\n${nextRelease.note}'
+    },
```

Comment:
I'm not familiar with this syntax. [Reading the semantic-release docs](https://semantic-release.gitbook.io/semantic-release/usage/plugins) makes me think this should be done more like
```
['@semantic-release/git', {
    message: 'Docs: ${nextRelease.version} [skip ci]\n\n${nextRelease.note}'
}]
```

Reply:
Good catch! You're right. I'll make the update.

Reply:
Updated!